packages/keyv/src/index.js

```diff
@@ -54,7 +54,7 @@ class Keyv extends EventEmitter {
 	}
 
 	_getKeyPrefix(key) {
-		return this.opts.namespace ? `${this.opts.namespace}:${key}` : key;
```

Comment:
@Jytesh can you confirm to me this is a typo? 🙂

Reply:
yes